app/views/api/v1/targets/index.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.targets @targets do |target|
+  json.partial! 'target', target:
```

Comment:
🌚 
```suggestion
  json.partial! 'target', target: target
```

Reply:
I this is a new syntax sugar and rubocop automatically change it